tests/ExcelProvider.Tests/ExcelProvider.Tests.fs

```diff
@@ -268,6 +269,8 @@ let ``Can load from multiple ranges``() =
     rows.[3].Fifth |> should equal null
     rows.[3].Sixth |> should equal null
 
+    rows |> Array.length |> should equal 4
```

Comment:
This was added to test the `Bounded` `Range` case not having extra rows. Having too few rows would throw on indexing on lines 265-270, as I found when making these changes.

Reply:
Maybe I should move this to the test I added... Hmm.